architecture/core_objects/openshift_model.adoc

```diff
@@ -67,84 +67,87 @@ OpenShift are immutable.
 
 Like Docker, images are retrieved within the scope of an image stream. The sample image below is from the `origin-ruby-sample` image stream, and can be retrieved by asking for the `ImageStreamImage` with name `origin-ruby-sample@sha256:4d3a646b58685449179a0c61ad4baa19a8df8ba668e0f0704b9ad16f5e16e642`.
 
+====
```

Comment:
It looks like the opening example blocks being introduced throughout this topic are missing their closing `====`s, which is messing up any section titles that follow.